src/main/webapp/src/components/sentimentAnalysisInfo.js

```diff
@@ -50,27 +50,35 @@ class SentimentAnalysisInfo extends React.Component {
     }
   }
 
+  /** 
+   * Get sentiment analysis info if it was sent before
+   * or load it instead.
+   */
   componentDidMount() {
```

Comment:
Since `componentDidMount` is a react method called at some point in the lifecycle, I'd rather make it call a separate method, where you can place your documentation. That will escalate better when/if you need to do more actions in the `componentDidMount` function.